docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md

```diff
@@ -854,7 +854,7 @@ export default Index;
 
 [View code in GitHub](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/pages/index.tsx)
 
-Once `summary` is available, it will create the `Flex` component with three `Box` components inside of it. `inventory_count`, `variant_count`, and `primary_category_name` will be populated with data returned from calling the `/catalog/summary` endpoint we added in [Add the Products endpoint](#add-the-products-endpoint). 
+`summary` creates the `Flex` component with three `Box` components inside of it. `inventory_count`, `variant_count`, and `primary_category_name` ware populated with data returned from calling the `/catalog/summary` endpoint added in [Add the Products endpoint](#add-the-products-endpoint). 
```

Comment:
`summary` creates the `Flex` component with three `Box` components inside of it. `inventory_count`, `variant_count`, and `primary_category_name` are populated with data returned from calling the `/catalog/summary` endpoint added in [Add the Products endpoint](#add-the-products-endpoint).